### ISSUE TITLE
client error

### ISSUE THREAD

**Issue by ardarico** [song940/node-dns#14]
Hi, 

I managed to run the server described in the README page, but I am in trouble with the client.

I have macOS 10.15.5, node 12.18.1.

mymac$ node client.js 
/.../path/.../client.js:12
dns.send(packet, function(err, res){
    ^

TypeError: dns.send is not a function
    at Object.<anonymous> (/.../path/.../client.js:12:5)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47

**Comment by song940** [comment]
Oh sorry, I forgot modify client examples in README, please see https://github.com/song940/node-dns/blob/master/example/index.js

And I‘ll correct it asap.

**Comment by ardarico** [comment]
I tried

const DNS = require('dns2');

const dns = new DNS();

(async () => {
  const result = await dns.resolveA('baidu.com')
  console.log(result);
  dns.close()
})();

but I got the following error:

mymac$ node client.js 
(node:52167) UnhandledPromiseRejectionWarning: TypeError: dns.resolveA is not a function
    at /..path../client.js:7:28
    at Object.<anonymous> (/..path../client.js:10:3)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47
(node:52167) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:52167) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
